contrib/segmentio/kafka.go.v0/kafka_test.go

```diff
@@ -0,0 +1,105 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package kafka
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+
+	kafka "github.com/segmentio/kafka-go"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	testGroupID = "gosegtest"
+	testTopic   = "gosegtest"
+)
+
+func skipIntegrationTest(t *testing.T) {
+	if _, ok := os.LookupEnv("INTEGRATION"); !ok {
+		t.Skip("🚧 Skipping integration test (INTEGRATION environment variable is not set)")
+	}
+}
+
+/*
+to run the integration test locally, update the broker name to localhost:29092:
+
+    docker network create segementio
+
+    docker run --rm \
+		--name zookeeper \
+		--network segementio \
+		-p 2181:2181 \
+		wurstmeister/zookeeper:3.4.6
+
+    docker run --rm \
+		--name kafka \
+		--network segementio \
+		-p 29092:29092 \
+		-e KAFKA_CREATE_TOPICS=gotest:1:1 \
+		-e KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181 \
+		-e KAFKA_LISTENERS=INSIDE://kafka:9092,OUTSIDE://kafka:29092 \
+		-e KAFKA_ADVERTISED_LISTENERS=INSIDE://kafka:9092,OUTSIDE://localhost:29092 \
+		-e KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT \
+		-e KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE \
+		wurstmeister/kafka:2.13-2.7.0
+*/
```

Comment:
I'm not able to get these instructions to work, and anyway we don't want users to have to edit the tests in order to run them locally. 

Let's either fix these instructions to bring up a kafka such that tests pass without modification (should be possible since 9092 is not a protected port) or remove these instructions.